From 1a7158fb9778ec669705041bf46358859e75f8b4 Mon Sep 17 00:00:00 2001
From: Peter Swain <peter.swain@marvell.com>
Date: Thu, 23 Aug 2018 08:47:02 -0700
Subject: [PATCH 0150/1051] spi: spi-cavium: missing line

dropped one line from previous commit
fbcf7844 "spi: spi-cavium: only one active Chip-Select"

Signed-off-by: Peter Swain <peter.swain@marvell.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 drivers/spi/spi-cavium.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/spi/spi-cavium.c b/drivers/spi/spi-cavium.c
index e01852f0ec72..19a9a2e6c60f 100644
--- a/drivers/spi/spi-cavium.c
+++ b/drivers/spi/spi-cavium.c
@@ -38,6 +38,7 @@ static int octeon_spi_do_transfer(struct octeon_spi *p,
 	bool cpha, cpol;
 	const u8 *tx_buf;
 	u8 *rx_buf;
+	int cs = msg->spi->chip_select;
 	u64 enax = 1ull << (12 + cs);
 	int len;
 	int i;
-- 
2.17.1

